app/component/map/tile-layer/DynamicParkingLots.js

```diff
@@ -59,6 +59,19 @@ class DynamicParkingLots {
       );
     });
 
+  getIcon = type => {
+    if (type === 'Parkhaus') {
+      return 'covered_carpark';
```

Comment:
Is Parkhaus and Tiefgarage both mapped to Parkhaus?

Reply:
Since we don't have an icon for Tiefgarage, I've mapped everything to Parkhaus in https://api.mih.mitfahren-bw.de/parkapi.json

This code doesn't take Tiefgarage into account.

Reply:
ParkAPI seems not to have a specified value set, but Tiefgarage is used by others. We should provide that type in ParkAPI and map it here to the same icon as Parkhaus.